ui-src/NVRApplication.js

```diff
@@ -0,0 +1,290 @@
+// vim: set et sw=2:
+//
+// TODO: test abort.
```

Comment:
// TODO: style: no globals? string literals? line length? fn comments?

I have removed this because:
* There are not really globals. What looks like globals are really scope confined to their respective modules (essentially each file's code is encapsulated in a function, given them all function scope). This would also be the case for everything put inside classes.
* string literals? Not sure what you meant here. I have standardized all code on strings in single quotes and, in a few places, use template strings (back-quoted). The eslint setup enforces this as well
* line length: I have used 80 and the prettifier I used always honors it, as far as I can tell
* fn comments: I have included JSDoc comments for all functions/methods

Reply:
"test abort" meant testing the scenario I mentioned where you switch the target URL (perhaps by clicking on another date) while a fetch is in flight. Sounds like it's still relevant.

sounds like we can remove the one about TODO. I was just reminding myself to, you know, actually read a Javascript style guide and try following one or two of the rules in it. :-/